src/routes/api.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+use Momento\LaravelExample\Controllers\WeatherController;
+use Illuminate\Support\Facades\Route;
+
+/*
+|--------------------------------------------------------------------------
+| API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register API routes for your application. These
+| routes are loaded by the RouteServiceProvider within a group which
+| is assigned the "api" middleware group. Enjoy building your API!
+|
+*/
+
+Route::get('/weather/{city}', [WeatherController::class, 'city']);
+Route::get('/weather/{zipcode}/{countryCode}', [WeatherController::class, 'zipcode']);
+Route::get('/weather/id/{cityId}', [WeatherController::class, 'cityId']);
```

Comment:
I had to copy these routes into my main `routes/api.php` for the app to work (before copying it was giving me a 404 Not Found for everything). Can you please take a look at that?